### PR TITLE
create success stories component

### PR DESCRIPTION
**Pull request by humailhasankhan**

### CODE REVIEW

File: src/pages/caseStudies.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import CaseStudyPreview from '../components/caseStudyPreview'
 import Button from '../components/button'
 import { navigate } from 'gatsby'
 import PropTypes from 'prop-types'
-import { CgArrowLongLeft } from 'react-icons/cg'
+import { FaRegArrowAltCircleLeft } from 'react-icons/fa'
 class CaseStudyIndex extends React.Component {
```
Comment (maham511) [inline src/pages/caseStudies.js:11]: Does line 10 class CaseStudyIndex need to be on new line (white space)? :)

File: src/pages/index.js [hunk 2]
```diff
@@ -3,26 +3,11 @@ import { graphql } from 'gatsby'
 import PropTypes from 'prop-types'
 import Layout from '../components/layout'
 import Home from '../components/home'
-import Button from '../components/button'
-import { navigate } from 'gatsby'
-import { CgArrowLongRight } from 'react-icons/cg'
 class RootIndex extends React.Component {
```
Comment (maham511) [inline src/pages/index.js:10]: White space here too?

File: src/templates/singleCaseStudy.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import get from 'lodash/get'
 import PropTypes from 'prop-types'
 import Button from '../components/button'
 import { navigate } from 'gatsby'
-import { CgArrowLongLeft } from 'react-icons/cg'
+import { FaRegArrowAltCircleLeft } from 'react-icons/fa'
 class CaseStudyTemplate extends React.Component {
```
Comment (maham511) [inline src/templates/singleCaseStudy.js:11]: And here?  :P

**Review by maham511 — CHANGES_REQUESTED** [review]

**Comment by pugoverflow** [comment]
Can we hold off on merging this please, not sure what it’s for, seems a bit over engineered to me @humailhasankhan

File: src/pages/index.js (hunk 2 above)
Comment (humailhasankhan) [reply src/pages/index.js:10]: You are right. For some reason there was white space but it was showing a timestamp for changes. When I looked at the line numbers, I realised there was no whitespace.

**Review by pugoverflow — APPROVED** [review]

**Review by maham511 — APPROVED** [review]

**Pull request merged**